views/js/recommended-modules.js

```diff
@@ -40,7 +40,10 @@ var mbo = {};
     toolbarLastElement: '.toolbar-icons a:last-of-type',
     recommendedModulesButton: '#recommended-modules-button',
     oldButton: '#page-header-desc-configuration-modules-list',
-    contentContainer: '#main-div .content-div .container:last',
+    contentContainer: [
+      '#main-div .content-div .container:last',
+      '#main-div .content-div .col:last',
+    ],
```

Comment:
```suggestion
    contentContainer: '#main-div .content-div .container:last, #main-div .content-div .js-content-col:last'
```

As it's temporary waiting for a new selector not related to the layout, and also I think we could just use a selector with `,` and select the first occurence of jQuery